### PR TITLE
Fixed some typos

### PR DESCRIPTION
**Pull request by hsbt**
Hi, I found typos and fixed them.

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -35,7 +35,7 @@
         Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean
         vestibulum, augue ut accumsan feugiat, mauris eros accumsan nunc,
         volutpat vulputate eros orci quis nulla. Cum sociis natoque penatibus
-        et magnis dis parturient montes, nascetur ridiculus mus. Sed varius
+        et magnis dis parturient montes, nascetur ridiculous mus. Sed varius
```
Comment (amatsuda) [inline spec/models/user_spec.rb:38]: This one is not an English word.

Comment (hsbt) [reply spec/models/user_spec.rb]: 👍 Thanks, It's my mistake.

**Comment by differentreality** [comment]
Thank you @hsbt :tada: good catch!

**Comment by differentreality** [comment]
@hsbt I think we should keep the change of your last commit, and please squash for merging

**Comment by hsbt** [comment]
@differentreality Thanks for squashing. I got your message now.

**Pull request merged**